src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+
+    public static void doWork() {
+        System.out.println("Truck started its work");
+
+    }
```

Comment:
The `doWork` method should not be static. By making it static, we cannot override it in subclasses, which is essential for polymorphism. Make sure to remove the `static` keyword and adjust the method accordingly.